plenum/test/req_handler/test_txn_author_agreement_handler.py

```diff
@@ -99,14 +69,53 @@ def test_update_state(txn_author_agreement_handler, taa_request):
     payload = get_payload_data(txn)
     text = payload[TXN_AUTHOR_AGREEMENT_TEXT]
     version = payload[TXN_AUTHOR_AGREEMENT_VERSION]
+    retired = payload.get(TXN_AUTHOR_AGREEMENT_RETIRED, False)
     digest = StaticTAAHelper.taa_digest(text, version)
     append_txn_metadata(txn, seq_no, txn_time, txn_id)
 
+    state_value = {TXN_AUTHOR_AGREEMENT_TEXT: text,
+                   TXN_AUTHOR_AGREEMENT_VERSION: version,
+                   TXN_AUTHOR_AGREEMENT_DIGEST: digest}
+    if retired:
```

Comment:
Fixture `taa_request` doesn't seem to be parametrized, why do we need this check here?